packages/core/src/accordion/accordion-panel.element.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { baseStyles, event, EventEmitter } from '@clr/core/internal';
+import { ClarityIcons, angleIcon } from '@clr/core/icon';
+import { html, LitElement, property } from 'lit-element';
+import { styles } from './accordion-panel.element.css.js';
+
+ClarityIcons.addIcons(angleIcon);
+
+/**
+ * Web component accordion panel to be used inside an accordion
+ *
+ * ```typescript
+ * import '@clr/core/accordion/register.js';
+ * ```
+ *
+ * ```html
+ * <cds-accordion>
+ *   <cds-accordion-panel expanded>
+ *     <cds-accordion-header>Item 1</cds-accordion-header>
+ *     <cds-accordion-content>Content 1</cds-accordion-content>
+ *   </cds-accordion-panel>
+ *   <cds-accordion-panel>
+ *     <cds-accordion-header>Item 2</cds-accordion-header>
+ *     <cds-accordion-content>Content 2</cds-accordion-content>
+ *   </cds-accordion-panel>
+ *   <cds-accordion-panel disabled>
+ *     <cds-accordion-header>Item 3</cds-accordion-header>
+ *     <cds-accordion-content>Content 3</cds-accordion-content>
+ *   </cds-accordion-panel>
+ * </cds-accordion>
+ * ```
+ *
+ * @element cds-accordion-panel
+ * @slot
+ * @slot cds-accordion-header
+ * @slot cds-accordion-content
+ * @event expandedChange - notify when the user has clicked the panel header
+ * @cssprop --border-color
+ * @cssprop --border-width
+ * @cssprop --color
+ * @cssprop --font-size
+ * @cssprop --font-weight
+ * @cssprop --header-background-color
```

Comment:
nit: using/assigning to `background` instead of `background-color` gives a bit more flexibility with theming